<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="QT-INBG-LT0089" failures="2" tests="10" name="Default test" time="0.112" errors="0" timestamp="2021-03-19T10:47:31 IST">
  <testcase classname="testNG.Clstwo" name="MobiledebitCardMethod1" time="0.0">
    <failure type="org.testng.TestNGException" message="
Method public void testNG.Clstwo.MobiledebitCardMethod1(java.lang.String,java.lang.String) requires a @DataProvider named : datasoruce">
      <![CDATA[org.testng.TestNGException: 
Method public void testNG.Clstwo.MobiledebitCardMethod1(java.lang.String,java.lang.String) requires a @DataProvider named : datasoruce
at org.testng.internal.Parameters.findDataProvider(Parameters.java:559)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- MobiledebitCardMethod1 -->
  <testcase classname="testNG.Clstwo" name="WebCreditcard1" time="0.002"/>
  <testcase classname="testNG.Clstwo" name="WebCreditcard2" time="0.003"/>
  <testcase classname="testNG.Clsthree" name="MobileDebitcard1" time="0.002"/>
  <testcase classname="testNG.Clsthree" name="MobileDebitcard2" time="0.002"/>
  <testcase classname="testNG.Clsone" name="ATest" time="0.0">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;URL&amp;apos; is required by @Test on method ATest but has not been marked @Optional or defined
in C:\Users\Qualitest\AppData\Local\Temp\testng-eclipse-269927652\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'URL' is required by @Test on method ATest but has not been marked @Optional or defined
in C:\Users\Qualitest\AppData\Local\Temp\testng-eclipse-269927652\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:268)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:356)
at org.testng.internal.Parameters.createParameters(Parameters.java:700)
at org.testng.internal.Parameters.handleParameters(Parameters.java:875)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ATest -->
  <testcase classname="testNG.Clsone" name="MobileCreditcard1" time="0.003"/>
  <testcase classname="testNG.Clsone" name="MobileCreditcard2" time="0.002"/>
  <testcase classname="testNG.Clsfour" name="WebDebitcard1" time="0.002"/>
  <testcase classname="testNG.Clsfour" name="WebDebitcard2" time="0.002"/>
</testsuite> <!-- Default test -->
